ci/zephyr_install.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -x
+
+export ZEPHYR_BASE="${HOME}/zephyr"
+
+ZEPHYR_SDK_RELEASE="0.11.0"
+ZEPHYR_SDK="zephyr-sdk-${ZEPHYR_SDK_RELEASE}-setup.run"
+ZEPHYR_SDK_URL="https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v${ZEPHYR_SDK_RELEASE}/${ZEPHYR_SDK}"
+
+CMAKE_RELEASE="3.16.3"
+CMAKE="cmake-${CMAKE_RELEASE}-Linux-x86_64.sh"
+CMAKE_URL="https://github.com/Kitware/CMake/releases/download/v${CMAKE_RELEASE}/${CMAKE}"
+
+export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+export ZEPHYR_SDK_INSTALL_DIR="${HOME}/zephyr-sdk"
+
+repos=(
+    "https://github.com/zephyrproject-rtos/zephyr"
```

Comment:
If you are going to use Zephyr, you really should use `west` to pull down the repos. How else are you going to be sure that you are pulling the right set?